tests/cli/test_clit.py

```diff
@@ -129,3 +131,24 @@ def test_execute_query_markdown_with_sql(self):
         """)
         self.assertIn(expected, stdout)
         self.assertIn('', stdout)
+
+    def test_execute_query_copy_query_result_to_clipboard(self):
+        sql = 'select Code, Name from country order by Code limit 3;'
+        commands = [
+            '\\cp',
+            sql
+        ]
+        stdout, stderr = self.process.communicate(commands_to_str(commands).encode())
+        stdout = stdout.decode('utf-8')
+        expected = dedent("""
+        +--------+-------------+
+        | Code   | Name        |
+        |--------+-------------|
+        | ABW    | Aruba       |
+        | AFG    | Afghanistan |
+        | AGO    | Angola      |
+        +--------+-------------+
+        """)[1:-1]
```

Comment:
This code is probably better.

```
        expected = dedent("""\
        +--------+-------------+
        | Code   | Name        |
        |--------+-------------|
        | ABW    | Aruba       |
        | AFG    | Afghanistan |
        | AGO    | Angola      |
        +--------+-------------+
        """)
```

Reply:
Hmm.
The test failed with the `"""\`. It seems that the output is different from the `[1:-1]`. 🤔